src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .split(';')
+    .filter((style) => style.length > 1 && style.includes(':'));
+
+  return styles.reduce((prev, style) => {
```

Comment:
It's not a `prev` value, it's an `accumulator`. Change it
```suggestion
  return styles.reduce((acc, style) => {
```